src/collectors/cephstats/cephstats.py

```diff
@@ -8,12 +8,38 @@
 import re
 import os
 import sys
-
 sys.path.insert(0, os.path.join(os.path.dirname(os.path.dirname(__file__)),
                                 'ceph'))
 from ceph import CephCollector
 
 
+# This is external to the CephCollector so it can be tested
+# separately.
+def process_ceph_status(output):
+    pattern = re.compile(r'\bclient io .*')
+    res = pattern.search(output)
+    if not res:
+        return {}
+    try:
+        ceph_stats = res.group()
+        number = re.compile(r'\d+')
+        rd = number.search(ceph_stats)
+        if rd:
+            wr = number.search(ceph_stats, rd.end())
+            if wr:
+                iops = number.search(ceph_stats, wr.end())
+        ret = {}
+        if rd:
+            ret['rd'] = rd.group()
+        if wr:
+            ret['wr'] = wr.group()
+        if iops:
+            ret['iops'] = iops.group()
+        return ret
+    except:
```

Comment:
Only thing that could possibly happen, but it seems that you're guarding against it, would be `re.search` returning `None`, then checking `.group()` which would raise an `AttributeError`.

But that case seems covered.